src/sentry.js

```diff
@@ -2,10 +2,7 @@ import * as Sentry from '@sentry/browser';
 
 Sentry.init({
   dsn: process.env.SENTRY_DSN,
-  integrations: [
-    Sentry.browserTracingIntegration(),
-    Sentry.replayIntegration()
-  ],
+  integrations: [Sentry.browserTracingIntegration()],
```

Comment:
we decided to remove the replays? It is a cool feature, but I guess too expensive?

Reply:
ok cool, yeah probably fine to leave it off for now, but cool to know we have it in our back pocket, could be useful some day in the future when we can figure out how to make it work